navapbc/labs-decision-support-tool
# DST-271 feat: generate llm result

## Ticket

https://navalabs.atlassian.net/browse/DST-271

## Changes
- added litellm call to prompt message

## Testing
- run `make start`
- navigate to localhost:8000/chat
- type in a question, response should be in prompt format
